app-ios/Modules/Sources/Session/SessionViewModel.swift

```diff
@@ -1,10 +1,37 @@
+import Dependencies
+import Event
 import Foundation
 import shared
 
-final class SessionViewModel {
+@MainActor
+final class SessionViewModel: ObservableObject {
+    @Dependency(\.eventKitClient) var eventKitClient
+    @Published var isAddingToCalendarConfirming: Bool = false
```

Comment:
This is nits! I think this flag represents dialog presenting state, so `@State` might be better 👀 
How about this?

Reply:
My understanding is that a `@State` on an `ObservableObject` doesn't update any views.

Reply:
@woxtu 
I didn't have enough to say. My suggestion is to use `@State` in `View` and use that 🙏🏼

Reply:
The view model takes its binding?

Reply:
Okay.